README.md

```diff
@@ -67,11 +67,11 @@ In order to keep Alamofire focused specifically on core networking implementatio
 - [Alamofire 2.0 Migration Guide](https://github.com/Alamofire/Alamofire/blob/master/Documentation/Alamofire%202.0%20Migration%20Guide.md)
 
 ## Communication
-
-- If you **need help**, use [Stack Overflow](https://stackoverflow.com/questions/tagged/alamofire). (Tag 'alamofire')
-- If you'd like to **ask a general question**, use [Stack Overflow](https://stackoverflow.com/questions/tagged/alamofire).
-- If you **found a bug**, open an issue.
-- If you **have a feature request**, open an issue.
+- If you need to **find or understand an API**, check [our documentation](http://alamofire.github.io/Alamofire/) or [Apple's documentation for `URLSession`](https://developer.apple.com/documentation/foundation/url_loading_system), on top of which Alamofire is built.
+- If you need **help with an Alamofire feature**, use [our forum on swift.org](https://forums.swift.org/c/related-projects/alamofire).
```

Comment:
I'm not sure the Swift Forum is a better place than Stack Overflow for users to get help. There's no gamified model on the Swift Forums to encourage more advanced users to help less experienced users.

Reply:
I think I need to call out still seeking help there, and that our forum is just for help and discussion of Alamofire specific things. I don't think users are getting the Alamofire specific help they need from SO, as most tagged questions don't get much in the way of response. Most of them don't really have anything to do with Alamofire anyway, so it's not much of an issue, but I think having actual AF related discussion on the forum will be better.

I'll add a first point about seeking general help from SO.